docs/usage/validators.md

```diff
@@ -349,16 +378,13 @@ print(DemoModel(ts='2017-11-08T14:00'))
 #> ts=datetime.datetime(2017, 11, 8, 14, 0)
 ```
 
-You'll often want to use this together with `pre`, since otherwise with `always=True`
-Pydantic would try to validate the default `None` which would cause an error.
-
 ## Reuse validators
 
-Occasionally, you will want to use the same validator on multiple fields/models (e.g. to
-normalize some input data). The "naive" approach would be to write a separate function,
-then call it from multiple decorators.  Obviously, this entails a lot of repetition and
-boiler plate code. To circumvent this, the `allow_reuse` parameter has been added to
-`pydantic.validator` in **v1.2** (`False` by default):
+Occasionally, you will want to use the same validator on multiple fields or models (e.g. to
+normalize some input data).
+
+In Pydantic V2, you can simply define a validation function once, then reuse it with the
+`field_validator` function, specifying the field and validation function.
```

Comment:
Perhaps worth demonstrating how to reuse validators in `mode=before|after`?

For ex:
```py

def first_char(v) -> Optional[str]:
    if isinstance(v, str) and len(v) > 1:
        return v[0]
    return v

class ModelA(BaseModel):
    a: str

    validate_a = field_validator("a", mode="before")(first_char)


class ModeB(BaseModel):
    b: str = "B"
   
    validate_b = field_validator("b", mode="after")(first_char)
```

Reply:
Added this example